superx/information_extractors/branch_info_extractor.py

```diff
@@ -55,7 +63,10 @@ def get_zip_file_link(self):
             page = requests.get(self.current_super['branch_url'])
             web_scrapper = BeautifulSoup(page.content, 'html.parser')
         except requests.ConnectionError:
-            raise ConnectionError(f'Unable to retrieve zip file link for {self.current_super["store_name"]}')
+            #pylint: disable=raise-missing-from
```

Comment:
do you understand what this error is suggesting that you do instead?

Reply:
not really, i went over this error with @shaibrown5 5  i figured it was for a reason. @shaibrown5  look at what i added in these lines see if its ok